Assets/Scripts/NetworkClient.cs

```diff
@@ -273,4 +312,61 @@ public bool IsConnected()
     {
         return mainWebSocket != null && mainWebSocket.State == WebSocketState.Open;
     }
+
+    private Dictionary<string, string> GetURLQueryParameters()
```

Comment:
Can we call this `GetConnectionParameters`, to match the server naming scheme? "connection params" are the full set of items passed via the URL query string (not just the server hostname and port).